docs/configuration/overview.md

```diff
@@ -9,21 +9,18 @@ description: Learn how to configure Yazi.
 
 There are three configuration files for Yazi:
 
-- [`yazi.toml`](./yazi.md) - General configuration
-- [`keymap.toml`](./keymap.md) - Keybinds configuration
-- [`theme.toml`](./theme.md) - Color scheme configuration
+- [`yazi.toml`](./yazi.md) - General configuration.
+- [`keymap.toml`](./keymap.md) - Keybinds configuration.
+- [`theme.toml`](./theme.md) - Color scheme configuration.
 
-You can find the default configurations at: https://github.com/sxyazi/yazi/tree/main/yazi-config/preset.
+You can find the default configuration files at https://github.com/sxyazi/yazi/tree/main/yazi-config/preset.
 
-To override any of the defaults, begin by copying the appropriate file to following location:
+To override any of the defaults, begin by copying the appropriate file (from the directory linked above) to the Yazi configuration directory.
 
-- Unix: `~/.config/yazi/`
+- On Unix/Unix-like systems: `~/.config/yazi/`
 - Windows: `C:\Users\USERNAME\AppData\Roaming\yazi\config\`
 
-For example, to edit the keymaps, start by copying `keymap.toml` file (found [here](https://github.com/sxyazi/yazi/tree/main/yazi-config/preset)) to:
-
-- Unix: `~/.config/yazi/keymap.toml`
-- Windows: `C:\Users\USERNAME\AppData\Roaming\yazi\config\keymap.toml`
+For example, to edit keybindings, start by copying the [`keymap.toml` file](https://github.com/sxyazi/yazi/blob/main/yazi-config/preset/keymap.toml) to `~/.config/yazi/keymap.toml` (Unix/Unix-like) or `C:\Users\USERNAME\AppData\Roaming\yazi\config\keymap.toml` (Windows).
```

Comment:
```suggestion
For example, to edit keybindings, start by copying the [`keymap.toml` file](https://github.com/sxyazi/yazi/blob/main/yazi-config/preset/keymap.toml) to:

- Unix-like: `~/.config/yazi/keymap.toml`
- Windows: `C:\Users\USERNAME\AppData\Roaming\yazi\config\keymap.toml`
```

Reply:
It just makes it longer and requires you to re-read the previous sentence to understand what this list is displaying. I think putting it in the sentence works better. We can make a poll in the Discord! 😆

Reply:
I thought you just agreed!

Reply:
I mean the text of the platform name not the list lol

Reply:
![Screenshot 2024-01-11 at 12 46 17 (Arc)](https://github.com/yazi-rs/yazi-rs.github.io/assets/47499684/c71c80aa-f886-415e-858e-b7703196f919)

You said you wanted the first one ahah.

Reply:
![CleanShot 2024-01-12 at 01 49 49](https://github.com/yazi-rs/yazi-rs.github.io/assets/17523360/231a96f4-6f0a-4446-ba1d-6f1a6cd3be2f)

Reply:
Ah okay. We can do a list there I guess.